glean-core/android/src/main/java/mozilla/telemetry/glean/rust/LibGleanFFI.kt

```diff
@@ -67,7 +67,12 @@ internal interface LibGleanFFI : Library {
 
     // Glean top-level API
 
-    fun glean_initialize(cfg: FfiConfiguration): Long
+    fun glean_initialize(
```

Comment:
This will affect iOS. It will need to explicitly pass `nil, nil, 0`.
Would it make sense to have a duplicate of this method for use by Kotlin instead?

Reply:
Could we have an FFI with this explicitly for migration (e.g. `glean_initialize_migration`), so it would be clear that we can remove it and revert back to the single argument version for Kotlin once we don't need migration code anymore?

Reply:
Mh, Good point. Yes, it does.